qiskit_ibm_runtime/utils/noise_learner_result.py

```diff
@@ -185,6 +198,53 @@ def num_qubits(self) -> int:
         """
         return len(self.qubits)
 
+    def draw_map(
+        self,
+        backend: BackendV2,
+        coordinates: Optional[list[list[int]]] = None,
```

Comment:
to emphasize the (x,y) innards
```suggestion
        coordinates: Optional[list[tuple[int, int]]] = None,
```

Reply:
It's not obvious to me why coordinates is the only kw argument thats not kw-only.

Reply:
I think that the original reason (which btw is questionable) was that all that was coming after `*` was a plotly thing. For example, `plot_bgcolor` is the name of the field that needs to be changed in a plotly figure to change the background color. But this does not matter anymore now that:
- `plot_bgcolor` changed to `background_color`
- a few args sneaked their way in without being plotly standard fields (such as "radius").
I will just remove the `*`